cassandane/Cassandane/Test/NewTestUrl.pm

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/perl
+
+package Cassandane::Test::NewTestUrl;
+use strict;
+use warnings;
+
+use JSON;
+use LWP::UserAgent;
+
+use base qw(Cassandane::Unit::TestCase);
```

Comment:
We don't expect Cassandane modules to be in a place where perl needs to look for them, so you gotta `use lib '.';` before referencing them

```suggestion
use lib '.';
use base qw(Cassandane::Unit::TestCase);
```

Reply:
Do you need use lib though? These files are pulled in by other cassandane files which are run by testrunner which has already set up the lib paths at that point?

Reply:
You do if you want to 'perl -c' all the modules before you run it, which I do, to make sure it isn't gonna trip over compilation failures halfway through (and is how I found it was missing)

Reply:
Okay, I've added that